service/src/main/java/io/camunda/service/ElementServices.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service;
+
+import io.camunda.search.clients.CamundaSearchClient;
+import io.camunda.service.security.auth.Authentication;
+import io.camunda.service.transformers.ServiceTransformers;
+import io.camunda.zeebe.broker.client.api.BrokerClient;
+import io.camunda.zeebe.gateway.impl.broker.request.BrokerSetVariablesRequest;
+import io.camunda.zeebe.protocol.impl.record.value.variable.VariableDocumentRecord;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class ElementServices extends ApiServices<ElementServices> {
```

Comment:
🔧 Should we rename this to `ElementInstanceServices` as well to reflect the change? Makes it a bit more relatable as well since "Element" could be anything 😅